flow/envs/base.py

```diff
@@ -214,7 +214,14 @@ def __init__(self,
             # render a frame
             self.render(reset=True)
         elif self.sim_params.render in [True, False]:
-            pass  # default to sumo-gui (if True) or sumo (if False)
+            # default to sumo-gui (if True) or sumo (if False)
+            if (self.sim_params.render is True) and self.sim_params.save_render:
+                path = os.path.expanduser('~')+'/flow_rendering/'
+                dir_time = time.strftime("%Y-%m-%d-%H%M%S")
+                if not os.path.exists(path):
+                    os.mkdir(path)
+                self.path = path + dir_time
```

Comment:
Think it might be worthwhile to add the exp_title to the path so that it's a little easier to track what movie you want to look at

Reply:
Good catch. Is the experiment path visible to an environment?

Reply:
Hmm, it appears it is not but it's worth adding. If we don't get it in in this PR could you add it as an issue?

Reply:
Looks like experiment titles are being passed as network names, so I was able to work with that.